containers-sig-laws/containers-sig-laws.cabal

```diff
@@ -0,0 +1,50 @@
+name:                containers-sig-laws
+version:             0.0.0
+synopsis:            Property tests for Map laws
+description:         Property tests for Map laws
+homepage:            https://github.com/kowainik/containers-backpack
+bug-reports:         https://github.com/kowainik/containers-backpack/issues
+license:             MPL-2.0
+license-file:        LICENSE
+author:              Kowainik
+maintainer:          xrom.xkov@gmail.com
+copyright:           2018 Kowainik
+category:            Backpack, Data, Containers
+build-type:          Simple
+extra-doc-files:     CHANGELOG.md
+cabal-version:       2.0
+tested-with:         GHC == 8.4.3
+
+source-repository head
+  type:                git
+  location:            https://github.com/kowainik/containers-backpack.git
+
+executable property-test-exe
+  hs-source-dirs:      app
+  main-is:             Main.hs
+  build-depends:       base
+                     , QuickCheck
+                     , containers-int-strict
+                     , containers-ordered-strict
+                     , containers-unordered-strict
+                     , containers-primitive
+                     , containers-contrib
+                     , containers-contrib-readonly
+
+  mixins:              containers-contrib (Map.Contrib.Group as Map.Contrib.Group.Int)
+                                 requires (Map as Map.Int)
+                     , containers-contrib (Map.Contrib.Group as Map.Contrib.Group.Ord)
+                                 requires (Map as Map.Ord)
+                     , containers-contrib (Map.Contrib.Group as Map.Contrib.Group.Hash)
```

Comment:
Do you really need to specify `Group` modules from the `containers-contrib` package in the `mixins` field? 🤔 
Even if you don't use them here?